@xen-orchestra/web/src/components/backups/jobs/panel/cards/BackupJobInfosCard.vue

```diff
@@ -18,17 +18,15 @@
           <VtsCopyButton :value="backupJob.id" />
         </template>
       </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('mode') }}</template>
+      <VtsCardRowKeyValue v-for="(mode, index) in backupJobModes" :key="mode">
+        <template #key>
+          <template v-if="index === 0">{{ t('mode', backupJobModes.length) }}</template>
+        </template>
         <template #value>
-          <UiTagsList>
-            <UiTag v-for="(mode, index) in backupJobModes" :key="index" accent="info" variant="secondary">
-              {{ mode }}
-            </UiTag>
-          </UiTagsList>
+          <span class="text-ellipsis">{{ mode }}</span>
         </template>
-        <template v-if="backupJobModes !== undefined" #addons>
-          <VtsCopyButton :value="backupJobModes.join(', ')" />
+        <template #addons>
+          <VtsCopyButton :value="mode" />
         </template>
       </VtsCardRowKeyValue>
     </div>
```

Comment:
When there are no modes maybe we should do that ? 
<img width="386" height="156" alt="image" src="https://github.com/user-attachments/assets/0d5ae70a-cb05-4ff6-a059-6c5cdbcad866" />
```suggestion
      </VtsCardRowKeyValue>
     </template>
     <VtsCardRowKeyValue v-else>
          <template #key>{{ t('mode', 2) }}</template>
    </VtsCardRowKeyValue>
    </div>
```

Reply:
As discussed during the meeting, this case shouldn’t happen once [this bug](https://github.com/vatesfr/xen-orchestra/pull/9210) is fixed.